searx/static/themes/simple/src/js/main/keyboard.js

```diff
@@ -66,13 +66,13 @@ searxng.ready(function () {
     'ArrowLeft': {
       key: '&#11013;',
       fun: highlightResult('up'),
-      des: 'Use left arrow to select previous search result',
+      des: 'left arrow to select previous search result',
```

Comment:
```suggestion
      des: 'select previous search result',
```
This one and the one after are the only key bindings where we mention the name of the key ("left arrow"), I think we should just drop that, even though we have two different bindings for that in vim mode.

Reply:
> This one and the one after are the only key bindings where we mention the name of the key ("left arrow"),

yes may its better to remove the name of the key / I added it because the symbol arrows were hard to see ... but maybe I just need a new pair of glasses :)

> even though we have two different bindings for that in vim mode.

I will remove the ArrowKey bindings from the vim table / thanks for pointing out :+1:

Reply:
You can use `ArrowLeft` instead of →, it's longer but more clear.

Reply:
@Bnyro, @BernieHuang2008: thanks for you hints, I fixed the issues you mentioned, see [diff](https://github.com/searxng/searxng/compare/ec19d4f502f47dc4292ec9ca5d7e0a7a96b5e15c..5c68aeee7311496b1d971f356a99e0c7fbbc0538) .. (resolve conversation).